framework_tvb/tvb/config/init/datatypes_registry.py

```diff
@@ -98,6 +98,11 @@
 
 # an alternative approach is to make each h5file declare if it has a corresponding datatype
 # then in a metaclass hook each class creation and populate a map
+from tvb.tests.framework.test_datatype import DummyDataType
+from tvb.tests.framework.test_datatype_h5 import DummyDataTypeH5
+from tvb.tests.framework.test_datatype_index import DummyDataTypeIndex
```

Comment:
What is this import doing here?
No imports from test in the core code modules pls!